reconcile/utils/terraform_client.py

```diff
@@ -151,44 +151,53 @@ def plan(self, enable_deletion):
             self.created_users.extend(created_users)
         return disabled_deletions_detected, errors
 
-    @retry()
```

Comment:
@maorfr Do you perhaps remember why this retry was added? I'd assume if there was an API error, i.e. AWS terraform itself would handle this with retries/timeouts. Thus if there was an error during terraform plan my assumption would be that it's more substantial and retries wouldn't matter.

Reply:
git blame for the rescue: #1289